### PR TITLE
Fix rails 5 2 for v1

### PR DESCRIPTION
**Pull request by emaglio**
- Limit trailblazer version to maximum 2.0.9 
- Fixed "Circular dependency detected" raised here: #66

### CODE REVIEW

File: trailblazer-rails.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "trailblazer", ">= 2.0.0", "< 2.1.0"
+  spec.add_dependency "trailblazer", "< 2.0.9"
```
Comment (seuros) [inline trailblazer-rails.gemspec]: we could have  "~> 2.0.9"

**Comment by seuros** [comment]
thank you @emaglio . You need to update the gemfile.lock in the others rails versions folders or they build will fail.

**Comment by emaglio** [comment]
@seuros fixed Gemfile.lock files and trailblazer version to `~> 2.0.0`.
We still have problem with rails3.2 that has failing tests (coudn't figure out why) and error in loading rails4.2 `SystemStackError: stack level too deep` (coudn't figure out why)

**Comment by apotonick** [comment]
TBH I think we can slowly drop Rails 3?

**Pull request merged**